templates/details.html

```diff
@@ -0,0 +1,116 @@
+            Da-i un comment lung de tot
+            Sa vezi scrollingul, pe bot,
+            Am si rima ca masina,
+            Varza rau, stricat ca INNA,
+            Trebe sa continui strofa
+            ca sa vad scrollingul
+            LALALALAL BATMANppppppp
+            Da-i un comment lung de tot
+            Sa vezi scrollingul, pe bot,
+            Am si rima ca masina,
+            Varza rau, stricat ca INNA,
+            Trebe sa continui strofa
+            ca sa vad scrollingul
+            LALALALAL BATMANppppppp
+            Da-i un comment lung de tot
+            Sa vezi scrollingul, pe bot,
+            Am si rima ca masina,
+            Varza rau, stricat ca INNA,
+            Trebe sa continui strofa
+            ca sa vad scrollingul
+            LALALALAL BATMANppppppp
+            Da-i un comment lung de tot
+            Sa vezi scrollingul, pe bot,
+            Am si rima ca masina,
+            Varza rau, stricat ca INNA,
+            Trebe sa continui strofa
+            ca sa vad scrollingul
+            LALALALAL BATMANppppppp
+            Da-i un comment lung de tot
+            Sa vezi scrollingul, pe bot,
+            Am si rima ca masina,
+            Varza rau, stricat ca INNA,
+            Trebe sa continui strofa
+            ca sa vad scrollingul
+            LALALALAL BATMANppppppp
+          </p>
+        </div>
+        <div class="review">
+          <h2 class="author">Gustav Molotov</h2>
+          <p class="comment">
+              Not enough room for my phat arse yet, but thanks to renault i lost about 3 kgs.Luckly i don't drink
+              anymore, am thinner and leading a happier life because of renault.Bad news is i sold two kidneys when
+              my clutch broke.
+          </p>
+        </div>
+        <div class="review">
+          <h2 class="author">Alex</h2>
+          <p class="comment">
+             I used to have money before purchasing this manufacturer.Unfortunately now i have my car but my
+             wife and kids left me because i couldn't further spend money in the house.Whenever a renault part
+             broke i had to access my kids' college fund to repair it.But overall, they work fine.4 stars from me.
+            </p>
+        </div>
+      </div>
+      <h2 class="overall-review">OVERALL RATING:</h2>
+      <fieldset class="rating">
+        <input type="radio" id="star5" name="rating" value="5" /><label for="star5" title="AMAZING!">5</label>
+        <input type="radio" id="star4" name="rating" value="4" /><label for="star4" title="NB BRAH">4</label>
+        <input type="radio" id="star3" name="rating" value="3" /><label for="star3" title="SO-SO">3 </label>
+        <input type="radio" id="star2" name="rating" value="2" /><label for="star2" title="LAME">2 </label>
+        <input type="radio" id="star1" name="rating" value="1" /><label for="star1" title="HORRIBLE">1</label>
+      </fieldset>
+    </div>
+  </div>
+{% endblock %}
+
+{% block footer %}
```

Comment:
You don't have to add the blocks in child template if you don't add anything in them